src/languages/elixir.js

```diff
@@ -223,6 +223,10 @@ export default function(hljs) {
       })
     ]
   };
+  const OPERATOR = {
+    scope: 'operator',
+    begin: /([-\|<]>)|(([\+\-\&\|]){2})|(<>)|([<>]=?)|([!=]==?)/,
```

Comment:
Could this be simplified or at the very least add a comment that lists out each one of the operators this is matching.

Reply:
`(([\+\-\&\|]){2})`

Is this even correct though?  Is `-|` or `&+` a valid operator?